### DISCUSSION TITLE
Commercial license

### DISCUSSION THREAD

**Discussion by coran21** [datalightinc/reliance-edge#40] [General]
Hello, 

I've been trying to reach somebody from your company for about 3 weeks to get pricing for the Reliance Edge commercial license. I filled in the form on your website, sent emails to both emails (info@tuxera.com and sales@tuxera.com), and called the Finland and USA phone numbers, but I was not able to reach anyone. Is there any way to reach somebody to get that pricing?

Thanks.

**Comment by jeremysherrill** [comment]
Hello Petr,

I checked with our sales department, and they indicated that Justin Engle reached out on December 20, 2023.  Would you please check your spam folder for an email from him?  Additionally, if you can provide your phone number, Justin will call you right away.

Thanks,
Jeremy

________________________________
From: Petr Hlineny ***@***.***>
Sent: Wednesday, January 3, 2024 2:15 PM
To: tuxera/reliance-edge ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [tuxera/reliance-edge] Commercial license (Discussion #40)


Hello,

I've been trying to reach somebody from your company for about 3 weeks to get pricing for the Reliance Edge commercial license. I filled in the form on your website, sent emails to both emails ***@***.******@***.***> and ***@***.******@***.***>), and called the Finland and USA phone numbers, but I was not able to reach anyone. Is there any way to reach somebody to get that pricing?

Thanks.

**Reply by coran21** [reply]
Hello, 

Thank you for the information. I sent the email again, and I got the response this time. I'll close this discussion because it doesn't look like I can delete it (our conversation probably doesn't help anybody).

Petr
